supportedBackends/local/src/main/scala/cromwell/backend/impl/local/LocalExecutor.scala

```diff
@@ -0,0 +1,165 @@
+package cromwell.backend.impl.local
+
+import java.nio.file.{FileSystem, Path, Paths}
+
+import com.typesafe.config.Config
+import cromwell.attributes.RuntimeAttributesValidation
+import cromwell.backend.BackendJobExecutionActor.{BackendJobExecutionFailedResponse, BackendJobExecutionResponse, BackendJobExecutionSucceededResponse}
+import cromwell.backend.{BackendJobDescriptor, BackendWorkflowDescriptor}
+import cromwell.core.CallContext
+import org.slf4j.LoggerFactory
+import wdl4s.WdlExpression
+import wdl4s.expression.ValueEvaluator
+import wdl4s.values.{WdlFile, WdlValue}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.language.postfixOps
+import scala.util.{Failure, Success, Try}
+
+case class LocalExecutor(workflowDescriptor: BackendWorkflowDescriptor,
+                    jobDescriptor: BackendJobDescriptor,
+                    filesystems: List[FileSystem],
+                    config: Config) extends SharedFileSystem with RuntimeAttributesValidation {
+  import better.files._
+
+  import scala.sys.process._
+
+  val logger = LoggerFactory.getLogger("LocalExecutor")
+
+  val call = jobDescriptor.key.call
+  val jobPaths = new JobPaths(workflowDescriptor, config, filesystems, jobDescriptor.key)
+  val callContext = new CallContext(jobPaths.callRoot.toString,
+    jobPaths.stdout.toAbsolutePath.toString,
+    jobPaths.stderr.toAbsolutePath.toString
+  )
+  val callEngineFunction = new LocalCallEngineFunctions(filesystems, callContext)
+  val evaluator = {
```

Comment:
This is the issue I'm referring to on BackendInitializationActor... to create an evaluator a jobDescriptor is needed :(.